server/parsec/api.py

```diff
@@ -74,7 +81,7 @@ def api(fn: ApiFnWithSelf) -> ApiFnWithSelf:  # pyright: ignore[reportMissingTyp
         case _:
             raise ValueError(f"unexpected family {m_family}")
     assert types["client_ctx"] == ForwardRef(expected_type_name), (
-        f"Expect `client_ctx` to be an `{expected_type_name}`"
+        f"Expect `client_ctx`({types['client_ctx']}) to be an `{expected_type_name}`"
```

Comment:
I was a bit stuck on that 
```
E   AssertionError: Expect `client_ctx`(<class 'parsec.client_context.AnonymousAccountClientContext'>) to be an `AnonymousAccountClientContext
````

because `from __future__ import annotations` was missing from account.py 

What do y'all think about adding that, as I'm quite sure it's not the first time I stumble on this
```suggestion
        f"Expect `client_ctx`({types['client_ctx']}) to be an `{expected_type_name}.` Did you add `from __future__ import annotations` ?"
```